RELEASE.md

```diff
@@ -5,6 +5,12 @@ Steps to release a new version of the Tempo Operator:
 1. Change the images tags to corresponding versions on `config/manager/controller_manager_config.yaml` the operator is usually aligned with the tempo versions. 
 1. Run `make bundle USER=os-observability OPERATOR_VERSION=x.y.z`, where `x.y.z` is the version that will be released
 1. Build, deploy and, run OpenShift tests locally against an OpenShift cluster `make e2e-openshift`. A locally installed [CRC](https://github.com/crc-org/crc) cluster can be used for testing.
+
+   Note: The e2e tests require MinIO (`make deploy-minio`) and opentelemetry-operator to be installed in the cluster.
```

Comment:
Maybe, should we add a target to install the OpenTelemetry Operator in the cluster?

Reply:
That'll depend if the test is done on OpenShift (then I'd suggest using OLM and install the RHOSDT data collection), otherwise the community version (either via manifests or olm).